packages/kit/src/runtime/client/client.js

```diff
@@ -682,10 +692,10 @@ async function load_node({ loader, parent, url, params, route, server_data_node
 							typeof data !== 'object'
 								? `a ${typeof data}`
 								: data instanceof Response
-									? 'a Response object'
-									: Array.isArray(data)
-										? 'an array'
-										: 'a non-plain object'
+								  ? 'a Response object'
+								  : Array.isArray(data)
+								    ? 'an array'
+								    : 'a non-plain object'
```

Comment:
looks like you'll need to `pnpm install` to get the latest version of Prettier before running `pnpm format` (annoying, I know)